### PR TITLE
Use woff2 instead ttf

### PR DESCRIPTION
**Pull request by nanot1m**
Reduces fonts size from 441kb to 202kb

https://caniuse.com/#feat=woff2
image

### CODE REVIEW

**Comment by vjeux** [comment]
Should we also update the font within the svg exported file?

**Comment by j-f1** [comment]
Probably shouldn’t remove the TTF since old exports will still refer to it, right?

**Comment by nanot1m** [comment]
> Should we also update the font within the svg exported file?

Sure, but it will not work until there would be a new release of excalidraw. We are using hardcoded urls, like
```
https://excalidraw.com/FG_Virgil.woff2
```

File: src/scene/export.ts [hunk 1]
```diff
@@ -94,11 +94,13 @@ export function exportToSvg(
...
         font-family: "Virgil";
-        src: url("https://excalidraw.com/FG_Virgil.ttf");
+        src: url("https://excalidraw.com/FG_Virgil.woff2") format("woff2"),
+             url("https://excalidraw.com/FG_Virgil.ttf") format("ttf");
       }
       @font-face {
         font-family: "Cascadia";
-        src: url("https://excalidraw.com/Cascadia.ttf");
+        src: url("https://excalidraw.com/Cascadia.woff2") format("woff2"),
+             url("https://excalidraw.com/Cascadia.ttf") format("ttf");
```
Comment (vjeux) [inline src/scene/export.ts:103]: Why do you keep both? Is there a reason why a fallback is useful?

Comment (nanot1m) [reply src/scene/export.ts]: Yes, was thinking about fallback. But it didn't work for svg file 😔 reverted this change

**Comment by nanot1m** [comment]
@vjeux I don't want to break stuff. 

As for me it would be better to update fonts in svg exported files after releasing current changes, because both ttf and woff2 will be available via direct link

**Comment by vjeux** [comment]
Our release process is really fast with `now`, both code and asset are live in under a minute. So we don't really need to worry about breaking changes that way.

**Pull request merged**